pkg/backend/display/progress_test.go

```diff
@@ -226,6 +243,65 @@ func TestProgressEvents(t *testing.T) {
 	}
 }
 
+func BenchmarkProgressEvents(t *testing.B) {
+	t.Setenv("TERM", "vt102")
```

Comment:
Why is this necessary in this test and not others?

Reply:
It's necessary for both tests that run progress tests for consistency across machines, as the rendering is TERM specific.

Reply:
OK. Is it not needed in other tests?

Reply:
No, they don't rely on this.

Reply:
OK!